xdp-dump/Makefile

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
+
+XDP_TARGETS  := xdpdump_bpf
+USER_TARGETS := xdpdump
+
+LIB_DIR   = ../lib
+CFLAGS    = -Werror
```

Comment:
BTW, this overrides the default cflags (-O2 -g) set in defines.mk - maybe it's better to add -Werror to those?

Reply:
Will try to do that, hope no other errors exist ;)

Reply:
Shouldn't be, but if you find any please fix them ;)

Reply:
Done will take care of this in the next commit